unittests/test_environments.py

```diff
@@ -256,6 +268,19 @@ def test_emit_loadenv_commands(base_environ, user_runtime,
     assert expected_commands == rt.emit_loadenv_commands(env0)
 
 
+def test_emit_loadenv_nomod_commands(base_environ, user_runtime, env0):
+    if test_util.has_sane_modules_system():
+        pytest.skip('test scenario not valid for a modules system')
```

Comment:
I don't think you need to skip the test as it's still valid when there is no modules system.